src/plugins/network.ts

```diff
@@ -93,6 +93,17 @@ export class NetworkManager {
     race.then(() => this._barrierPromises.delete(race));
   }
 
+  private _nullableFrameBarrier(req: Request): Frame | null {
```

Comment:
Changed in PW 1.38 where the error would be thrown when accessing frame that are not created.